### PR TITLE
ember-cli notes 13 july 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-07/july-13.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-07-13
+
+## Attendees
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-07/july-13.md]: https://github.com/ember-cli/ember-cli/pull/7019

Comment (kratiahuja) [inline ember-cli/2017-07/july-13.md]: https://github.com/ember-cli/rfcs/pull/108: Move to FCP?

Comment (hjdivad) [inline ember-cli/2017-07/july-13.md]: peer dependencies

Comment (hjdivad) [inline ember-cli/2017-07/july-13.md]: ember-cli-babel catch -> validate cc @rwjblue

**Comment by hjdivad** [comment]
cc @twokul @Turbo87 @kategengler @kellyselden @kratiahuja

**Comment by twokul** [comment]
lgtm

**Pull request merged**